.travis.yml

```diff
@@ -4,6 +4,8 @@ cache: pip
 python:
   - "3.6"
   - "3.7"
+  - "3.8"
+  - "3.9"
```

Comment:
we already ran out of credits so I wouldn't add more for now.

Reply:
tl;dr
Just an update on this credit problem, we can ask for more credits and I've applied for more monthly credits.

Details: 

As it turns out, travis was having performance problems because people were abusing their open-source service, so they made the open-source credits as opt-in.

We ran out of credits on travis-ci because we would now have to apply as open-source projects, they will verify we are indeed an open-source project and allow us more credits. :smile:

I've applied for 10 times more credit per month, let's see how much is allocated! :smile: